src/dags/wior.py

```diff
@@ -0,0 +1,248 @@
+import operator
+import requests
+from requests.auth import HTTPBasicAuth
+import json
+import logging
+
+from airflow import DAG
+from airflow.models import Variable
+from airflow.operators.bash_operator import BashOperator
+from airflow.operators.python_operator import PythonOperator
+from airflow.operators.postgres_operator import PostgresOperator
+
+from common.db import DatabaseEngine
+from environs import Env
+
+from ogr2ogr_operator import Ogr2OgrOperator
+from provenance_rename_operator import ProvenanceRenameOperator
+from pgcomparator_cdc_operator import PgComparatorCDCOperator
+from sqlalchemy_create_object_operator import SqlAlchemyCreateObjectOperator
+
+
+from common import (
+    default_args,
+    slack_webhook_token,
+    DATAPUNT_ENVIRONMENT,
+    SHARED_DIR,
+    MessageOperator,
+)
+
+from postgres_check_operator import (
+    PostgresMultiCheckOperator,
+    COUNT_CHECK,
+    GEO_CHECK,
+)
+
+from sql.wior import (
+    DROP_COLS,
+    SQL_DROP_TMP_TABLE,
+    SQL_GEOM_CONVERT,
+    SQL_ADD_PK,
+    SQL_SET_DATE_DATA_TYPES,
+)
+
+dag_id = "wior"
+variables = Variable.get(dag_id, deserialize_json=True)
+data_endpoint = variables["data_endpoints"]["wfs"]
+tmp_dir = f"{SHARED_DIR}/{dag_id}"
+data_file = f"{tmp_dir}/{dag_id}.geojson"
+env = Env()
+db_conn = DatabaseEngine()
+password = env("AIRFLOW_CONN_WIOR_PASSWD")
+user = env("AIRFLOW_CONN_WIOR_USER")
+base_url = env("AIRFLOW_CONN_WIOR_BASE_URL")
+total_checks = []
+count_checks = []
+geo_checks = []
+
+
+# define logger for output to console
+logger = logging.getLogger(__name__)
+
+
+# needed to put quotes on elements in geotypes for SQL_CHECK_GEO
```

Comment:
This function is appearing in a lot of DAGs. Better to put it in a central place, eg. the `common` module.

Reply:
Goeie, toevoegd en nu uit `common` gehaald